aws/resource_aws_dms_replication_task_test.go

```diff
@@ -27,6 +27,11 @@ func TestAccAWSDmsReplicationTaskBasic(t *testing.T) {
 					resource.TestCheckResourceAttrSet(resourceName, "replication_task_arn"),
 				),
 			},
+			{
+				Config:             dmsReplicationTaskConfig(randId),
+				PlanOnly:           true,
+				ExpectNonEmptyPlan: false,
+			},
```

Comment:
Now we run plan after create to assert that there are no changes. Without the fix, this will fail.